monai/apps/nnunet/nnunet_bundle.py

```
@@ -36,17 +36,17 @@
 
 
 def get_nnunet_trainer(
-    dataset_name_or_id: Union[str, int],
+    dataset_name_or_id: str | int,
     configuration: str,
-    fold: Union[int, str],
+    fold: int | str,
     trainer_class_name: str = "nnUNetTrainer",
     plans_identifier: str = "nnUNetPlans",
     use_compressed_data: bool = False,
```

Ruff (0.14.10)

Unused function argument: `use_compressed_data`

(ARG001)